uamqp/message.py

```diff
@@ -116,6 +120,42 @@ def __init__(self,
             self._annotations = annotations
             self._header = header
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        state["_body"] = list(state["_body"].data) if state["_body"].data else ""
```

Comment:
things will be tricky after I add the sequence type support.
I'll take care of this part.